frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/edgeScrolling.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useRef } from 'react';
+import { EdgeHoveringData, EdgeHoveringObserver } from './edgeHoveringObserver';
+
+type EdgeScrolling = {
+	start: (container: HTMLElement) => void;
+	stop: () => void;
+	isRunning: boolean;
+};
+type Options = {
+	edgeSize?: number,
+	speed?: number,
+};
+export const edgeScrolling = (options: Options = {}): EdgeScrolling => {
+	const { speed = 1000 } = options;
+	let scrollSpeed = 0;
+	let containerElement: HTMLElement | null = null;
+	let observer = new EdgeHoveringObserver();
+	let prevTime;
+
+	const handleMouseMove = ({ side, proximity }: EdgeHoveringData) => {
+		scrollSpeed = ((proximity / 100) ** 3) * speed;
+		if (side === 'left') {
+			scrollSpeed *= -1;
+		}
+	};
+
+	const scrollContainer = () => {
+		if (!observer.isObserving || !containerElement) return;
+
+		const now = new Date().getTime();
+		if (prevTime && scrollSpeed !== 0) {
+			const timeDiff = now - prevTime;
+			const distanceToScroll = scrollSpeed * (timeDiff / 1_000);
```

Comment:
Can we simply use 1000?
and change distanceToScroll to incrementX or scrollIncrement or somtehing of the sort, because with this name at least to me makes me think about that is the distance to the scroll edge or something similar, not that is the incrementation applied to the scroll.